packages/smooth_app/lib/cards/product_cards/product_title_card.dart

```diff
@@ -29,14 +30,27 @@ class ProductTitleCard extends StatelessWidget {
     final Widget trailingWidget;
     final String brands = product.brands ?? appLocalizations.unknownBrand;
     final String quantity = product.quantity ?? '';
+
     if (isRemovable && !isSelectable) {
       final ContinuousScanModel model = context.watch<ContinuousScanModel>();
       subtitleText = '$brands${quantity == '' ? '' : ', $quantity'}';
       trailingWidget = InkWell(
-        onTap: () async => model.removeBarcode(product.barcode!),
-        child: const Icon(
-          Icons.clear_rounded,
-          size: MINIMUM_TOUCH_SIZE,
+        customBorder: const CircleBorder(),
+        onTap: () async {
+          await model.removeBarcode(product.barcode!);
+
+          // Vibrate twice
+          SmoothHapticFeedback.confirm();
+        },
+        child: Tooltip(
+          message: appLocalizations.product_card_remove_product_tooltip,
+          child: const Padding(
+            padding: EdgeInsets.all(SMALL_SPACE),
+            child: Icon(
+              Icons.clear_rounded,
+              size: DEFAULT_ICON_SIZE,
```

Comment:
```suggestion
              size: MINIMUM_TOUCH_SIZE,
```

cf.
```dart
/// Standard minimum touch/target size, cf. goldens.dart
const double MINIMUM_TOUCH_SIZE = kMinInteractiveDimension;
```

Reply:
The reason of this change is that the button is way too big with that dimension =~ 2 lines of text.
I prefer to have instead a smaller one, but with bigger padding around.

Reply:
I understand the idea of a smaller button. The thing is that if we want to comply with standard accessibility, the minimum size of a target is MINIMUM_TOUCH_SIZE x MINIMUM_TOUCH_SIZE.

Reply:
Fair enough: MINIMUM_TOUCH_SIZE is 48, DEFAULT_ICON_SIZE is 24 and SMALL_SPACE is 12: we're good.
The icon will look smaller, but the tap area will be large enough (12 + 24 + 12).